# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1301349187 25200
#      Mon Mar 28 14:53:07 2011 -0700
# Node ID e4a22302dc97d7b53189d735f4b653d7941a9b99
# Parent  3ba6e702e6fd71a25d2c962df1667eab2085e2d0
Add do_braindead_shortcircuit_evaluation to the list of variables set by --traditional option.

diff --git a/doc/ChangeLog b/doc/ChangeLog
--- a/doc/ChangeLog
+++ b/doc/ChangeLog
@@ -1,8 +1,13 @@
+2011-03-28  Rik  <octave@nomad.inbox5.com>
+
+	* interpreter/basics.txi: Add do_braindead_shortcircuit_evaluation to
+	the list of variables set by --traditional option.
+
 2011-03-27  Rik  <octave@nomad.inbox5.com>
 
 	* interpreter/expr.txi: Add allow_noninteger_range_as_index to
 	documentation.
 
 2011-03-27  Rik  <octave@nomad.inbox5.com>
 
 	* interpreter/expr.txi: Add do_braindead_shortcircuit_evaluation to
diff --git a/doc/interpreter/basics.txi b/doc/interpreter/basics.txi
--- a/doc/interpreter/basics.txi
+++ b/doc/interpreter/basics.txi
@@ -210,16 +210,17 @@ user preferences to the following values
 @group
 PS1                             = ">> "
 PS2                             = ""
 allow_noninteger_range_as_index = true
 beep_on_error                   = true
 confirm_recursive_rmdir         = false
 crash_dumps_octave_core         = false
 default_save_options            = "-mat-binary"
+do_braindead_shortcircuit_evaluation = true
 fixed_point_format              = true
 history_timestamp_format_string = "%%-- %D %I:%M %p --%%"
 page_screen_output              = false
 print_empty_dimensions          = false
 @end group
 @end example
 
 @noindent
